tests/subaccounts/updateSubaccount.test.js

```diff
@@ -8,7 +8,7 @@ test('should update subaccount', async () => {
 
     let retrievedSubaccount = await client.subaccounts.retrieve(subaccount.id)
     expect(retrievedSubaccount.name).toBe('jefke')
-    expect(retrievedSubaccount.email).toBe(email)
+    expect(retrievedSubaccount.email).toBe(undefined)
```

Comment:
No need to assert that the e-mail address is undefined here; as far as the JS client is concernced, 'email' does not exist anymore on subaccounts.

Reply:
👍  I will get rid of subaccount email assertions